core/toolbox/toolbox.js

```diff
@@ -13,10 +13,9 @@
 goog.provide('Blockly.Toolbox');
 
 goog.require('Blockly.CollapsibleToolboxCategory');
-goog.require('Blockly.constants');
 goog.require('Blockly.Css');
```

Comment:
Add back `Blockly.constants`. It doesn't look like it but Blockly.TOOLBOX_AT_TOP comes from this file.

Reply:
I'm not sure why it got removed. I'll add it back